src/renderer/views/Popular/Popular.js

```diff
@@ -2,49 +2,61 @@ import Vue from 'vue'
 import FtLoader from '../../components/ft-loader/ft-loader.vue'
 import FtCard from '../../components/ft-card/ft-card.vue'
 import FtElementList from '../../components/ft-element-list/ft-element-list.vue'
+import FtIconButton from '../../components/ft-icon-button/ft-icon-button.vue'
 
 export default Vue.extend({
   name: 'Popular',
   components: {
     'ft-loader': FtLoader,
     'ft-card': FtCard,
-    'ft-element-list': FtElementList
+    'ft-element-list': FtElementList,
+    'ft-icon-button': FtIconButton
   },
   data: function () {
     return {
       isLoading: false,
       shownResults: []
     }
   },
+  computed: {
+    popularCache: function () {
+      return this.$store.getters.getPopularCache
+    }
+  },
   mounted: function () {
-    this.getTrendingInfo()
+    this.shownResults = this.popularCache
+    if (!this.shownResults || this.shownResults.length < 1) {
+      this.fetchTrendingInfo()
+    }
   },
   methods: {
-    getTrendingInfo: function () {
-      this.isLoading = true
-
+    refreshTrendingInfo: function () {
+      this.fetchTrendingInfo()
+    },
+    fetchTrendingInfo: async function () {
```

Comment:
Please rename `fetchTrendingInfo()` to `fetchPopularInfo()` as to not confuse with the Trending page.